GOVERNANCE.md

```diff
@@ -8,25 +10,84 @@ This document defines the project governance for Contour.
 
 ## Code Repositories
 
-The following code repositories are governed by Contour community and
-maintained under the `ProjectContour` namespace.
+All the code repositories owned by the Project Contour Github Organization are
+governed by Contour community and maintained under the `ProjectContour` namespace.
```

Comment:
Why the CamelCase here?

Reply:
I was like that before, I suppose it was for easy reading!